setup.py

```diff
@@ -36,6 +36,7 @@
     "importlib_metadata",
     "pywebpush>=1.13.0",
     "py-vapid>=1.7.0",
+    "matrix-common==1.0.0",
```

Comment:
I don't remember if we had a conversation, but 👍 on pinning this. I think we want exact versions here since we're just trying to avoid vendoring this code.

Reply:
I seem to remember someone telling me to pin the `matrix-common` import, but can't remember where that was.